doc/development/guidelines.md

```diff
@@ -259,22 +259,51 @@ Changelog = "https://github.com/openalea/pkg_name/releases"
-For most cases, we recommend hosting the data directly in the pkg source directory. However, if the size of data files becomes large (e.g. for realistic images or databases) they will bloat your source directory and slow down the installation and/or CI workflow. 
-In such cases, we recommend storing you data on public spaces ([GitHub](https://github.com) or [Zenodo](https://zenodo.org/)), and provide access in a dedicated module using [Pooch](https://www.fatiando.org/pooch). Both approaches are detailed below.
+You might want to include resources/data files (i.e. any files that is not a *.py file) in your package for a number of reasons, that will determine your packaging strategy:
+- the resource is mandatory for your python source code to run (eg a dll or a template file) => include it as a package resource (see 'Distributing as a package resource' below)
+- the resource is mandatory for your python source code to run, but is too large in size for github (eg a trained deep-learning model) => refer it as an external resource in the code (see 'Distributing as an external resource' below)
+- the resource is only used in tests (and is relatively small in size) => add a test/data dir and access it with relative path (see 'Direct access using relative paths' bellow)
+- the resource is only used in notebooks (and is relatively small in size) => add a doc/examples/data dir and access it with relative path (see 'Direct access using relative paths' bellow)
+- the resource is accessed by test and notebooks (and is relatively small in size) => include it as a package resource (see 'Distributing as a package resource' below)
+- the resource is accessed by test, notebooks, and/or is usefull for other packages (eg database), and/or is composed of large file (e.g. phenotyping images) => refer it as an external resource in the code (see 'Distributing as an external resource' below)
 
-### Distributing data with the package sources
+### Direct access using relative paths
+
+This method is recommended only if you are sure that the data files will be accessed by cloning the git repository (e.g. clone/run tests locally or on CI runners, clone/build doc locally or by ReadTheDoc runner).
+It can also work for data located in source dir (e.g src/openalea/pkg_name/data), but it is not recommended, as some build tools will zip the src dir and defeat the method. In the later case prefer 'Distributing as a package resource' below)
+
+This method is pretty straightforward, as you just have to add a data dir, e.g. for test:
+
+```bash
+pkg_name
+├── ...
+├── test
+│   ├── test_A.py                        
+│   ├── data
+│       ├── data_file_A.csv
+```
+
+and in test_A.py, access the data file with:
+
+```python
+from pathlib import Path
+this_dir = Path(__file__).parent if '__file__' in globals() else Path(".").resolve()
+data_dir = this_dir.parent / "data" 
+data_A_path = data_dir / "data_file_A.csv"
```

Comment:
What is the strategy for several data files?
A specific function that return the file with doc or an automatic strategy that list all the files and produce the python module data.py?